### ISSUE TITLE
Clearing input field does not reset date value

### ISSUE THREAD

**Issue by octronic** [soulim/ember-cli-bootstrap-datepicker#33]
When a date is has been selected with the date picker and I manually clear the input field, the date is not reset. If I bound the datepicker value to a property on a model, the property still has the selected date as value (although the input field is empty).

Resetting the property with the 'clear'-button works correctly.

**Comment by soulim** [comment]
This might be related to the way `bootstrap-datepicker` handles changes in the input field. Will take a look.

**Comment by octronic** [comment]
Thanks!

Currently I wrap your component in an extra component and use

```
inputChanged: Ember.on('input', function () { 
  //check input and reset date
})
```

as workaround.

**Comment by ismyrnow** [comment]
Any chance that PR might be merged? I'm dealing with this issue at the moment as well.

**Comment by jjgillespie** [comment]
@soulim Are there any updates on this issue or any reasons not to merge PR 35?

**Comment by jjgillespie** [comment]
Great, thanks! And thanks for the great component.

**Comment by soulim** [comment]
Just merged #35. Sorry for delay. I didn't have enough time to give more love to this project.

**Comment by soulim** [comment]
Thank you for being awesome and patient!
